functions/src/index.ts

```diff
@@ -107,3 +114,13 @@ async function handleUpdateDocument(
     logs.error(e as Error)
   }
 }
+
+/**
+ * addSearchableFields
+ */
```

Comment:
Compared to the other function comments, maybe this one is a bit less complete

Reply:
Oopsy I forgot this one